packages/gene-tools/src/generators/component-generator/utils/resolveDynamicOptions.ts

```diff
@@ -4,7 +4,7 @@ import { resolveGeneratorOptions } from './resolveGeneratorOptions';
 export type ComponentPromptsProfileType = 'basic' | 'sample' | 'advanced';
 
 export interface ComponentSchemaType {
-  ['prompts-profile']: ComponentPromptsProfileType;
+  promptsProfile: ComponentPromptsProfileType;
```

Comment:
NX started parsing schema props to camelCase so they were missed since one of NX updates